packages/amplify_core/lib/src/types/api/graphql/graphql_request.dart

```diff
@@ -22,6 +22,9 @@ class GraphQLRequest<T> {
   /// Only required if your backend has multiple GraphQL endpoints in the amplifyconfiguration.dart file. This parameter is then needed to specify which one to use for this request.
   final String? apiName;
 
+  /// A map of Strings to dynamically use for custom headers in the http request.
+  final Map<String, String>? customHeaders;
```

Comment:
What other kind of headers are there? ;-) I would just call em `headers`

Reply:
Good point, updated!